cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java

```diff
@@ -233,7 +233,8 @@ public LoadMonitorState state(OperationProgress operationProgress, MetadataClien
                                         validPartitionRatio,
                                         numValidPartitions,
                                         totalNumPartitions,
-                                        extrapolations);
+                                        extrapolations,
+                                        _loadMonitorTaskRunner.reasonOfLatestPauseOrResume());
       case SAMPLING:
```

Comment:
should we also add the reason information for `SAMPLING` state?

Reply:
Reason is only relevant for `RUNNING -> PAUSED` and `PAUSED -> RUNNING` transitions. Is there a case that I am missing?